02-oop-inheritance/js/script.js

```diff
@@ -65,7 +68,32 @@ rocky.unsubscribeObserver(person2);
 var DownloadableMovie = function () {};
 
 DownloadableMovie.prototype = Movie.prototype;
-DownloadableMovie.prototype.download = function () {console.log('downloading')};
+DownloadableMovie.prototype.download = function () {
+    console.log('downloading');
+};
 
 var downloadableMovie = new DownloadableMovie();
 downloadableMovie.download();
+
+function extend (Movie, SocialMixin) {
```

Comment:
Also this function must be generic not specific for movies and social mixin only